operations/power_management/System_Power_Off_Procedures.md

```diff
@@ -17,22 +17,30 @@ To make sure that the system is healthy before power off and all the information
 
 To shut down compute nodes and User Access Nodes \(UANs\), refer to [Shut Down and Power Off Compute and User Access Nodes](Shut_Down_and_Power_Off_Compute_and_User_Access_Nodes.md).
 
+## Power Off the External File Systems
+
+To power off the external Lustre file system (ClusterStor), refer to [Power Off the External Lustre File System](Power_Off_the_External_Lustre_File_System.md).
+
+To power off the external SpectrumScale (GPFS) file system, refer to site procedures.
```

Comment:
If you agree that the product name of this contains a space, as in Spectrum Scale, then global find/replace. I will leave the remaining instances without comment.

Reply:
Fixed in all remaining spots.